packages/buckram/assets/styles/components/toc/_generic.scss

```diff
@@ -219,13 +219,7 @@
 }
 
 @if $type == 'prince' {
-  h3.front-matter-number,
```

Comment:
This is not needed anymore https://www.princexml.com/doc/pdf-bookmarks/#bookmark-levels because we are not using a title tag